apps/web/src/hooks/useRegisterNameCallback.ts

```diff
@@ -85,6 +99,35 @@ export function useRegisterNameCallback(
   const normalizedName = normalizeEnsDomainName(name);
   const isDiscounted = Boolean(discountKey && validationData);
 
+  const prepareReverseRecordSignature = useCallback(async () => {
```

Comment:
[nit] Since this function also handles making the sign request, I think a better name could be
`signMessageForReverseRecord` or `getSignatureForReverseRecord`